### PR TITLE
Replace pkg_resources with importlib.resources

### PR DESCRIPTION
**Pull request by cdce8p**
`pkg_resources` is deprecated. Replace it with `importlib.resources`.
https://docs.python.org/3/library/importlib.resources.html

### CODE REVIEW

**Comment by cdce8p** [comment]
> it seems this does not work in python < 3.12, is this expected?

It does now. Apparently the API for `imporlib.resources` has been a bit unstable since `3.8`. My default is `3.13` and as I started writing this PR it was quite short with `read_text` until I saw the errors with older Python versions 🤦🏻

File: velbusaio/handler.py [hunk 1]
```diff
@@ -53,10 +54,18 @@ def __init__(
...
+            self.broadcast = json.loads(data)
+        else:
+            async with async_open(
+                str(
+                    importlib.resources.files(__name__.split(".")[0]).joinpath(
+                        "module_spec/broadcast.json"
+                    )
+                )
+            ) as protocol_file:
+                self.broadcast = json.loads(await protocol_file.read())
```
Comment (cereal2nd) [inline velbusaio/handler.py:68]: read_text is this handled correct in the event loop?
This is not throwing a warning?

Comment (cdce8p) [reply velbusaio/handler.py:68]: Technically, `read_text` is blocking as it just delegates to `open`. Not sure it matters much but I can change it to use `async_open` instead.

File: pyproject.toml [hunk 2]
```diff
@@ -27,7 +27,7 @@ classifiers = [
     "Topic :: Software Development :: Libraries",
     "Topic :: Software Development :: Libraries :: Python Modules",
 ]
-requires-python = ">=3.8.0"
+requires-python = ">=3.9.0"
```
Comment (cdce8p) [inline pyproject.toml:30]: Just saw that `importlib.resources.files` was only added in `3.9`.
https://docs.python.org/3.13/library/importlib.resources.html#importlib.resources.files

**Pull request merged**